compileopts/config.go

```diff
@@ -367,6 +367,22 @@ func (c *Config) LLVMFeatures() string {
 	return c.Options.LLVMFeatures
 }
 
+func (c *Config) InterpPackage() bool {
```

Comment:
Can you add some short documentation to these methods? Something like "InterpPackage returns whether the interp package should be run per package, according to the -interp-pass flag".

Reply:
Done.